test/recipe/admin/locales/locale.en.json

```diff
@@ -7,7 +7,7 @@
     "Allow_locale_customization_and_language_support": "Allow locale customization and language support",
     "Allow_select_from_given_options_using_buttons": "Allow select from given options using buttons",
     "Allow_select_from_given_options_using_drow_down": "Allow select from given options using drow down",
-    "All_ui_Specs_are_listed_here": "All UI Specs are listed here",
+    "All_ui_specs_are_listed_here": "All UI Specs are listed here",
```

Comment:
i think this change causes below glitch

<img width="406" height="400" alt="Image" src="https://github.com/user-attachments/assets/cfcc6a0d-f433-400d-b140-4be488a30b99" />